src/main/java/gregtech/api/util/GTToolTypes.java

```diff
@@ -0,0 +1,42 @@
+package gregtech.api.util;
+
+import gregtech.api.items.toolitem.ToolMetaItem;
+import gregtech.common.items.MetaItems;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class is used for handling GT tool types instead of specific tools
+ */
+public class GTToolTypes {
```

Comment:
Could these be moved into GTValues maybe? Or is that too early to call these (i.e., they end up null)?